admin/class-primary-term-admin.php

```diff
@@ -274,22 +312,31 @@ private function map_taxonomies_for_js( $taxonomy ) {
 			'singularLabel' => $taxonomy->labels->singular_name,
 			'fieldId'       => $this->generate_field_id( $taxonomy->name ),
 			'restBase'      => ( $taxonomy->rest_base ) ? $taxonomy->rest_base : $taxonomy->name,
-			'terms'         => array_map( array( $this, 'map_terms_for_js' ), $terms ),
+			'terms'         => $this->map_terms_for_js( $terms ),
 		);
 	}
 
 	/**
-	 * Returns an array suitable for use in the javascript.
+	 * Returns an array suitable for use in the JavaScript.
 	 *
-	 * @param stdClass $term The term to map.
+	 * @param array $terms The terms to map.
 	 *
 	 * @return array The mapped terms.
 	 */
-	private function map_terms_for_js( $term ) {
-		return array(
-			'id'   => $term->term_id,
-			'name' => $term->name,
-		);
+	private function map_terms_for_js( $terms ) {
```

Comment:
This was used for an array map, now the behaviour has been changed. I would recommand to undo this to make this method more readable.